Grace.py

```diff
@@ -0,0 +1,40 @@
+#A task was given before to just include a name and then parse it of which I did 
+#The second assignment was to parse both firstname and lastname together which was
+#done and Favour Samuel was gotten.
+#The third task isto Capitalise the first letters of "hello amazing grace"
+
+#Code for the second task.
+import argparse
+
+from argparse import ArgumentParser
+
+
+def hello(firstname, Lastname):
+    print( f'Hello {firstname} {Lastname}')
+
+
+if __name__ == '__main__':
+    args_to_parse = ArgumentParser()
+    args_to_parse.add_argument('--firstname', required=True, help='The name to greet')
+    args_to_parse.add_argument('--Lastname', required=True, help='The name to greet')
+    args = args_to_parse.parse_args()
+
+    hello(args.firstname,  args.Lastname)
+    
+#Code for the third task
+def hello(firstname, Lastname):
+    print( f'Hello {firstname} {Lastname}')
+
+
+if __name__ == '__main__':
+    args_to_parse = ArgumentParser()
+    args_to_parse.add_argument('--firstname', required=True, help='The name to greet')
+    args_to_parse.add_argument('--Lastname', required=True, help='The name to greet')
+    args = args_to_parse.parse_args()
+
+    hello(args.firstname.title(),  args.Lastname.title())

```

Comment:
I suggest moving `title()` to `hello`. The reason is, that the `hello` function is responsible for display. Or in other words: try to keep things together.

Reply:
I tried doing this but it keeps giving me error. I do not know where to place "hello()" to make it work like you suggested.

Reply:
No need to replace `hello`. But have it like:

```
def hello(firstname, lastname):
    print(f'Hello { firstname.title() } { lastname.title() }')
```

Reply:
sorry, I meant title(). I have made the changes. It works fine.